### PR TITLE
docs: fix js dependencies in webpack example

### PR DESCRIPTION
**Pull request by dfguo**
example won't run unless these modules are installed.

### CODE REVIEW

File: examples/browser-webpack/package.json [hunk 1]
```diff
@@ -11,13 +11,17 @@
     "babel-core": "^6.24.1",
     "babel-loader": "^7.0.0",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
```
Comment (dryajov) [inline examples/browser-webpack/package.json]: Wha does the `babel-preset-stage-0` do and why do we need it here?

File: examples/browser-webpack/package.json [hunk 2]
```diff
@@ -11,13 +11,17 @@
...
     "json-loader": "^0.5.4",
-    "react": "^15.5.4",
     "react-hot-loader": "^1.3.1",
     "webpack": "^2.5.1",
     "webpack-dev-server": "^2.4.5"
   },
   "dependencies": {
+    "ipfs": "^0.26.0",
```
Comment (dryajov) [inline examples/browser-webpack/package.json]: We still want to use IPFS from this repo -  it needs to be something like:

```
"dependencies": {
   "ipfs": "../../../src/core",
   ....
}
```

Comment (mkg20001) [reply examples/browser-webpack/package.json]: This one should be correct
```
"dependencies": {
   "ipfs": "file:../../",
}
```

File: examples/browser-webpack/package.json (hunk 1 above)
Comment (dfguo) [reply examples/browser-webpack/package.json]: I'm not sure why it was included in the `.babelrc` https://github.com/ipfs/js-ipfs/blob/master/examples/browser-webpack/.babelrc

File: examples/browser-webpack/package.json (hunk 2 above)
Comment (diasdavid) [reply examples/browser-webpack/package.json]: Oh, nice! Much better

File: examples/browser-webpack/src/components/app.js [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 const React = require('react')
 const Buffer = require('safe-buffer').Buffer
-const IPFS = require('../../../../src/core') // replace this by line below
-// const IPFS = require('ipfs')
+// const IPFS = require('../../../../src/core') // replace this by line below
```
Comment (diasdavid) [inline examples/browser-webpack/src/components/app.js]: You can remove this line now.

**Comment by diasdavid** [comment]
@dryajov can you investigate and test if these changes are necessary? I've already run the examples just fine.

**Comment by dryajov** [comment]
looking into it!

**Pull request merged**